float32 x
float32 y
float32 tolerance
string turtle_name
---
bool success
